Source/EasyNetQ.Management.Client.Tests/EasyNetQ.Management.Client.Tests.csproj

```diff
@@ -19,6 +20,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="Json\Consumers.json">

```

Comment:
You can write

```csharp
<EmbeddedResource Include="Json\*.json">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
</EmbeddedResource>
```

and delete all these lines.

These
```
 <ItemGroup>
    <None Remove="Json\Bindings.json" />
    <None Remove="Json\Channels.json" />
    <None Remove="Json\Connections.json" />
    <None Remove="Json\Consumers.json" />
    <None Remove="Json\Definitions.json" />
    <None Remove="Json\Exchanges.json" />
    <None Remove="Json\Federations.json" />
    <None Remove="Json\Nodes.json" />
    <None Remove="Json\Overview.json" />
    <None Remove="Json\Policies_ha.json" />
    <None Remove="Json\Queues.json" />
  </ItemGroup>
```
can be deleted too.

Reply:
Resolved with latest commit